python/pyproject.toml

```diff
@@ -63,6 +63,7 @@ python-snappy = { version = "^0.6.1", optional = true }
 thrift = { version = "^0.16.0", optional = true }
 
 s3fs = { version = "2022.8.2", optional = true }
+boto3 = "1.24.59"
```

Comment:
Could you make this one optional? Similar to Hive. Because not everyone uses Glue or Boto, this makes the package quite heavy to install.